csrc/ir/interface_nodes.h

```diff
@@ -282,6 +282,9 @@ class NVF_API TensorView : public Val {
       int64_t position,
       ComputeAtMode mode = ComputeAtMode::Standard);
 
+  //! Create a new broadcast IterDoimain with extent one in the loop domain
```

Comment:
Let's say this is a 0-dim tensor. Would inserting a broadcast result in any semantic change? 

Asserting non 0-dim would be also reasonable.

Reply:
I think the "semantic change" is "we want to schedule it like a 1D broadcasting tensor"? Allowing 0-dim is aligned with PyTorch's `unsqueeze`:

```python
>>> torch.zeros(()).unsqueeze(0).shape
torch.Size([1])
>>> torch.zeros(()).unsqueeze(-1).shape
torch.Size([1])
```